gui/index2.html

```diff
@@ -35,8 +35,8 @@
 		<block type="wait"></block>
 	</xml>
 	<!-- END HTML -->
-	<script src="./static/js/interface.js"></script>
-	<script src="./static/js/view.js"></script>
+	<!--<script src="./static/js/interface.js"></script>-->
```

Comment:
These are interfering with functions in React

Reply:
Can delete if unnecessary